discussions/no-customizing-classnames.md

```diff
@@ -1,58 +0,0 @@
-# No Customizing `className`'s
```

Comment:
It definitely does, I just don't want to confuse people with "no classNames" article and then exposing classNames everywhere without explaining Fela bits.